xml/Microsoft.Extensions.Hosting/IHostApplicationBuilder.xml

```diff
@@ -1,166 +1,171 @@
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig newslot virtual instance void ConfigureContainer&lt;TContainerBuilder&gt;(class Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1&lt;!!TContainerBuilder&gt; factory, class System.Action`1&lt;!!TContainerBuilder&gt; configure) cil managed" />
+      <MemberSignature Language="DocId" Value="M:Microsoft.Extensions.Hosting.IHostApplicationBuilder.ConfigureContainer``1(Microsoft.Extensions.DependencyInjection.IServiceProviderFactory{``0},System.Action{``0})" />
+      <MemberSignature Language="VB.NET" Value="Public Sub ConfigureContainer(Of TContainerBuilder) (factory As IServiceProviderFactory(Of TContainerBuilder), Optional configure As Action(Of TContainerBuilder) = Nothing)" />
+      <MemberSignature Language="F#" Value="abstract member ConfigureContainer : Microsoft.Extensions.DependencyInjection.IServiceProviderFactory&lt;'ContainerBuilder&gt; * Action&lt;'ContainerBuilder&gt; -&gt; unit" Usage="iHostApplicationBuilder.ConfigureContainer (factory, configure)" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Extensions.Hosting.Abstractions</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <TypeParameters>
+        <TypeParameter Name="TContainerBuilder" />
+      </TypeParameters>
+      <Parameters>
+        <Parameter Name="factory" Type="Microsoft.Extensions.DependencyInjection.IServiceProviderFactory&lt;TContainerBuilder&gt;" />
+        <Parameter Name="configure" Type="System.Action&lt;TContainerBuilder&gt;">
+          <Attributes>
+            <Attribute>
+              <AttributeName Language="C#">[System.Runtime.CompilerServices.Nullable(new System.Byte[] { 2, 1 })]</AttributeName>
+              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(new System.Byte[] { 2, 1 })&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+        </Parameter>
+      </Parameters>
+      <Docs>
+        <typeparam name="TContainerBuilder">The type of builder provided by the <see cref="T:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1" />.</typeparam>
+        <param name="factory">The factory object that can create the <typeparamref name="TContainerBuilder" /> and <see cref="T:System.IServiceProvider" />.</param>
+        <param name="configure">A delegate used to configure the <typeparamref T="TContainerBuilder" />. This can be used to configure services using APIS specific to the <see cref="T:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1" /> implementation.</param>

```

Comment:
```suggestion
        <param name="configure">A delegate used to configure the <typeparamref T="TContainerBuilder" />. This can be used to configure services using APIs specific to the <see cref="T:Microsoft.Extensions.DependencyInjection.IServiceProviderFactory`1" /> implementation.</param>
```